src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import logo from "../../../public/assets/logo.svg";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function Navbar() {
+    return (
+        <div
+            className='navbar bg-base-100'
+            style={{
```

Comment:
shouldn't the style be done with tailwind?? or at leas in a style sheet? not inline!!